lib/evm_database.rb

```diff
@@ -18,6 +18,8 @@ class EvmDatabase
     VmdbDatabase
   }
 
+  EXTRA_CLASSES = %w(MiqAeDatastore)
```

Comment:
If Automate is becoming a Rails Engine - should we rename `EXTRA_CLASSES` to `RAILS_ENGINE_MODEL_CLASS_NAMES` or something like that?  /cc @Fryguy